code/modules/clothing/masks/gasmask.dm

```diff
@@ -20,6 +20,24 @@
 		"Plasmaman" = 'icons/mob/species/plasmaman/mask.dmi'
 		)
 
+/obj/item/clothing/mask/gas/wirecutter_act(mob/living/user, obj/item/I)
+	if(type != /obj/item/clothing/mask/gas)
+		return
```

Comment:
```suggestion
	if(!istype(src, /obj/item/clothing/mask/gas))
		return
```

Reply:
`istype()` would also allow subtypes through as far as I know, which includes stuff like welding gas masks, syndicate masks, clown masks, etc.

Reply:
Valid, ignore this suggestion